python/lsst/ts/m2/mock_command.py

```diff
@@ -107,30 +200,62 @@ async def apply_forces(self, message, message_event):
         ----------
         message : `dict`
             Command message.
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
         message_event : `MockMessageEvent`
             Instance of MockMessageEvent to write the event.
+
+        Returns
+        -------
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
+        `CommandStatus`
+            Status of command execution.
         """
 
-        # TODO (DM-30851):
-        # Implement in DM-30851
-        pass
+        model.apply_forces(message["axial"], message["tangent"])
+
+        return model, CommandStatus.Success
 
-    async def position_mirror(self, message, message_event):
+    async def position_mirror(self, message, model, message_event):
         """Position the mirror.
 
         Parameters
         ----------
         message : `dict`
             Command message.
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
         message_event : `MockMessageEvent`
             Instance of MockMessageEvent to write the event.
+
+        Returns
+        -------
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
+        `CommandStatus`
+            Status of command execution.
         """
 
-        # TODO (DM-30851):
-        # Implement in DM-30851
-        pass
+        mirror_position_set_point = {
+            "x": message["x"],
+            "y": message["y"],
+            "z": message["z"],
+            "xRot": message["xRot"],
+            "yRot": message["yRot"],
+            "zRot": message["zRot"],
+        }
+
+        try:
+            command_success = model.handle_position_mirror(mirror_position_set_point)
+        except RuntimeError:
+            command_success = False
 
-    async def reset_force_offsets(self, message, message_event):
+        await message_event.write_m2_assembly_in_position(command_success)
+
+        return model, CommandStatus.Success if command_success else CommandStatus.Fail
```

Comment:
Same comment as before. Consider doing `command_sucess == True`

Reply:
Done in the commit: e99800d. Thanks!